### PR TITLE
[10.0][MIG] purchase_order_line_sequence

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Superseeds https://github.com/OCA/purchase-workflow/pull/520.

### CODE REVIEW

**Comment by MiquelRForgeFlow** [comment]
@aheficent please review

**Review by JordiBForgeFlow — APPROVED** [review]
:+1: Functional review

File: purchase_order_line_sequence/README.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+=========================================
+Purchase order lines with sequence number
+=========================================
+
+Provides a new field sequence on the purchase order, which allows to manage the order of its purchase order lines.
+The sequence then is propagated to the Invoice lines and the Stock Moves.
```
Comment (AaronHForgeFlow) [inline purchase_order_line_sequence/README.rst:10]: The sequence is not propagated to the stock moves, only to the invoice.

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]

File: purchase_order_line_sequence/README.rst (hunk 1 above)
Comment (MiquelRForgeFlow) [reply purchase_order_line_sequence/README.rst:10]: You should see in the <code>Initial Demand</code> tab

**Comment by MiquelRForgeFlow** [comment]
@aheficent update review please

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Tested on runbot

**Pull request merged**